### PR TITLE
Update M5 release notes to reflect changes in `junitPlatformTest` tas…

### PR DESCRIPTION
**Pull request by mkobit**
…k in Gradle plugin

issue: https://github.com/junit-team/junit5/issues/708

## Overview

This is to close out on the additional items required in https://github.com/junit-team/junit5/pull/751

---

I hereby agree to the terms of the JUnit Contributor License Agreement.

---

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Method [preconditions](http://junit.org/junit5/docs/snapshot/api/org/junit/platform/commons/util/Preconditions.html) are checked and documented in the method's Javadoc
- [x] [Coding conventions](https://github.com/junit-team/junit5/blob/master/CONTRIBUTING.md#coding-conventions) (e.g. for logging) have been followed
- [x] Change is covered by [automated tests](https://github.com/junit-team/junit5/blob/master/CONTRIBUTING.md#tests)
- [x] Public API has [Javadoc](https://github.com/junit-team/junit5/blob/master/CONTRIBUTING.md#javadoc) and [`@API` annotations](http://junit.org/junit5/docs/snapshot/api/org/junit/platform/commons/meta/API.html)
- [x] Change is documented in the [User Guide](http://junit.org/junit5/docs/snapshot/user-guide/) and [Release Notes](http://junit.org/junit5/docs/snapshot/user-guide/#release-notes)
- [x] All [continuous integration builds](https://github.com/junit-team/junit5#continuous-integration-builds) pass

### CODE REVIEW

File: documentation/src/docs/asciidoc/release-notes-5.0.0-M5.adoc [hunk 1]
```diff
@@ -27,6 +27,8 @@ on GitHub.
 
 ===== New Features and Improvements
 
+* The JUnit Platform Gradle plugin task `junitPlatformTest` can be accessed
```
Comment (sbrannen) [inline documentation/src/docs/asciidoc/release-notes-5.0.0-M5.adoc]: Slight modification: "The JUnit Platform Gradle plugin task `junitPlatformTest` can now be accessed directly during the configuration phase of a build."

**Comment by sbrannen** [comment]
Oops.... the latest build for your PR broke.

Please run `gradle spotlessApply` and then squash your changes and force push.

Thanks!

**Comment by mkobit** [comment]
My fault on that one! I tried the "edit from GitHub" feature for that and thought "seems trivial, what could go wrong?"

Lesson learned (again). Always build the code first!

**Comment by sbrannen** [comment]
> My fault on that one! I tried the "edit from GitHub" feature for that and thought "seems trivial, what could go wrong?"

😆 

> Lesson learned (again). Always build the code first!

Indeed. Spotless gets the best of all of us at times.

Thanks for fixing the formatting.

**Comment by sbrannen** [comment]
This has been merged into `master`.

Cheers

**Pull request merged**